code/project.clj

```diff
@@ -88,7 +88,10 @@
                                [org.apache.curator/curator-test]]
               :resource-paths ["test-resources"]
               :env            {:nuvla-session-key "test-resources/session.key"
-                               :nuvla-session-crt "test-resources/session.crt"}
+                               :nuvla-session-crt "test-resources/session.crt"
+                               :es-sniffer-init "no"
+                               :kafka-producer-init "yes"
+                               :kafka-client-conf-client-id "test-nuvla-server"}
```

Comment:
What is this client-id for ?

Reply:
When Kafka client (producer or consumer) contacts Kafka server, it needs to specify its ID (from docu: "The sole purpose of this is to be able to track the source of requests beyond just ip and port by allowing a logical application name to be included in Kafka logs and monitoring aggregates.") There is a default in the code which is "nuvla-server". Here I simply want to explicitly see in the logs that this is the test client and validate the propagation of the environment variables to the Kafka client.